### PR TITLE
error checking in the init data migration

### PR DESCRIPTION
**Pull request by kave**
@rosskarchner 
@jimmynotjim

### CODE REVIEW

**Comment by jimmynotjim** [comment]
It looks good, but I'd like @rosskarchner to sign off.

**Comment by rosskarchner** [comment]
The logic looks fine-- but I'm wondering if high-numbered migrations work the way we're assuming they do?

If the numbers define simple order, then that's great. But if they are expected to be a *sequence*, then we might be shooting ourselves in the foot. For example, on a server where migration 9999 has already been run, and we deploy a new release that include hypothetical migration "1234", will it be ignored because it's less than 9999?

**Comment by rosskarchner** [comment]
I guess we can test that by creating a migration smaller than 9999, running `migrate`, and seeing if it gets picked up?

**Comment by kave** [comment]
@rosskarchner hmm yes i tested this scenario and your right... hmm i'll try to think of a solution

File: cfgov/v1/migrations/9999_init_data.py [hunk 1]
```diff
@@ -15,24 +14,28 @@
...
 
         # Events Landing Page required for event `import-data` command
-        events = EventLandingPage(title='Events', slug='events')
-        site_root.add_child(instance=events)
-        revision = events.save_revision(
-            user=admin_user,
-            submitted_for_moderation=False,
-        )
-        revision.publish()
+        if not EventLandingPage.objects.filter(title='Events').exists():
```
Comment (kurtw) [inline cfgov/v1/migrations/9999_init_data.py]: The `EventLandingPage` type is deprecated in the BrowseFilterablePage PR. It's a normal `BrowsePage` now.

**Comment by kave** [comment]
Closing this PR in favor of https://github.com/cfpb/cfgov-refresh/pull/1357

**Pull request closed without merging**